docker/images/spark/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+FROM liaojiayidocker/hadoop-base:latest
+
+ENV SPARK_VERSION=2.4.5
+ENV SPARK_HADOOP_VERSION=2.7
+ENV ICEBERG_VERSION=0.8.0
```

Comment:
`-incubating` is part of the version, so you probably want to include it here. The 0.9.0 release will not have `-incubating` in the version because the project graduated from the incubator.